app/assets/javascripts/application.js

```diff
@@ -36,6 +36,10 @@ function onContentLoad() {
   // highlight tabs with errors
   tfm.tools.highlightTabErrors();
 
+  $('body').popover({selector: 'a[rel="popover"]'});
```

Comment:
This changed from the previous version which was `$('a[rel="popover"]').popover();`